app/controllers/teams_controller.rb

```diff
@@ -65,7 +65,7 @@ def add_member
 
     respond_to do |format|
       if @team.update!
-        format.html { redirect_to team_path(@team), :notice => "#{new_member.username} was successfully added to #{@team.name}" }
+        format.html { redirect_to team_path(@team), :notice => t(".controller.add-member", member:new_member.username, team:@team.name) }
```

Comment:
This is just a style check... I'm suggesting it because I haven't reconfigured RuboCop again to make autocorrections just yet 😅 

```suggestion
        format.html { redirect_to team_path(@team), :notice => t(".controller.add-member", :member => new_member.username, :team => @team.name) }
```